### DISCUSSION TITLE
Vitodens 300-W mit Vitotronic H02C

### DISCUSSION THREAD

**Discussion by ghost** [openv/openv#543] [General]
Schönen guten Tag,

Ich versuche mich gerade zum Ersten Mal an einer Auslesung bestimmter Werte meiner Heizung und hänge an dem Punkt was ich mit meiner Vitodens 300-W (B3HB-26) mit einer Vitotronic Steuerung H02C in der Vcontrold.xml und Vito.xml eingeben muss
ID = ?
Protocol = ?
Ich habe es mit 20CB und 300,KW versucht, dies wird allerdings als undefiniert wiedergegeben. 

Ebenso kann ich nicht via vclient eine Abfrage starten, weil mir das auch nicht zugelassen wird.^
TTY Net: No connection to 127.0.0.1:3002
No connection to host 127.0.0.1 on port 3002

Hier ein Auszug aus meiner vcontrold.xml

![image](https://user-images.githubusercontent.com/104017748/164038801-02048eca-5c3c-4865-84da-38e2b44653ce.png)

Ich wäre dankbar für Tipps oder Anstöße in die richtige Richtung. 

Vielen lieben Dank
Jörg

**Comment by mattmeye** [comment]
Hallo Jörg,
bist du inzwischen erfolgreich geworden? Habe auch eine 300-W und kriege noch keine Verbindung.

**Reply by ghost** [reply]
Hallo Matthias,

ja, das läuft seit ca. einem 3/4 Jahr spitze. Wie kann ich dir helfen? VG Jörg

**Reply by mattmeye** [reply]
Ich hab jetzt auch eine Verbindung. Ich habe zunächst eine Handvoll von Adressen ausgelesen, die ich für interessant hielt. Einige ändern sich davon aber bei mir wohl nicht (Speicherladepumpe, Nachladeunterdrückung, Speichertemperatur 2) bzw. habe noch keine Erklärung für die Werte gefunden (SolarInfo 4).

**Reply by 42asdff** [reply]
Hallo Jörg,

ich habe auch eine 300-W und bin gerade ziemlich lost. 
Wäre super, wenn Du mir auf die Sprünge helfen könntest.

Mein Stand:
- Kommuniktation mit vcontrold steht, telnet kann grundsätzlich Werte abfragen

Meine Fragen:
- Wie komme ich an meine ID?
- Wo finde ich eine Vito.xml deren Datenpunkte zu meiner 300-W passt, bzw. gibts eine schritt für schritt anleitung für die erstellung derselben?
Vielen lieben Dank
